src/main/scala/Configs.scala

```diff
@@ -153,6 +153,7 @@ class DefaultConfig extends Config (
       }
       case BuildRoCC => Nil
       case RoccNMemChannels => site(BuildRoCC).map(_.nMemChannels).foldLeft(0)(_ + _)
+      case RoccNPTWPorts => site(BuildRoCC).map(_.nPTWPorts).foldLeft(0)(_ + _)
```

Comment:
It seems that Java and Scala favor using consistent casing for acronyms, even in camelcase (e.g. package xml, class XMLParser.  So RoCC is fine.